source/includes/steps-atlas-terraform-file-from-template.yaml

```diff
@@ -0,0 +1,242 @@
+         documentation:
+
+         - :atlas:`Amazon Web Services </reference/amazon-aws/>`
+         - :atlas:`Google Cloud Platform </reference/google-gcp/>`
+         - :atlas:`Microsoft Azure </reference/microsoft-azure/>`
+
+     * - ``mongodbatlas_cluster``
+       - ``provider_instance_size_name``
+       - Instance size of the |service| cluster that this configuration
+         creates.
+
+         Either:
+
+         - Choose one of the Shared Tier instance sizes included in the
+           template: ``M2`` or ``M5``.
+         - Enter another instance size that |service| supports for your
+           chosen provider and region.
+
+         For details about the instance sizes that each provider and 
+         region supports, see the following sections in the |service| 
+         documentation:
+
+         - :atlas:`Amazon Web Services </reference/amazon-aws/>`
+         - :atlas:`Google Cloud Platform </reference/google-gcp/>`
+         - :atlas:`Microsoft Azure </reference/microsoft-azure/>`
+
+     * - ``mongodbatlas_cluster``
+       - ``disk_size_gbs``
+       - Disk size of the |service| cluster that this configuration
+         creates. Ensure that you provide a value that is equal to or 
+         less than the maximum disk size for the instance size you 
+         chose.
+
+         - For ``M2`` clusters, enter ``2``. 
+         - For ``M5`` clusters, enter ``5``.
+
+         For details about the disk sizes that each provider supports 
+         for each instance size, see the following sections in the 
+         |service| documentation:
+
+         - :atlas:`Amazon Web Services </reference/amazon-aws/>`
+         - :atlas:`Google Cloud Platform </reference/google-gcp/>`
+         - :atlas:`Microsoft Azure </reference/microsoft-azure/>`
+  
+---
+title: "Update the ``local`` variables."
+level: 4
+ref: update-local-variables
+content: |
+
+  .. warning::
+
+     The ``local`` variables contain sensitive information. **Do not** 
+     check these values in to a repository that is available publicly.
+
+  Provide values for the following ``local`` variables:
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 50 50
+
+     * - Variable
+       - Value
+
+     * - ``mongodb_atlas_api_pub_key``
+       - |service| public API key.
+
+     * - ``mongodb_atlas_api_pri_key``
+       - |service| private API key.
+
+     * - ``mongodb_atlas_org_id``
+       - |service| organization ID in which you want to create a
+         project.
+
+     * - ``mongodb_atlas_database_username``
+       - Username of the MongoDB database user that |service| creates
+         for your cluster. 
+
+     * - ``mongodb_atlas_database_user_password``
+       - Password for the MongoDB database user named in
+         ``mongodb_atlas_database_username``.
+
+     * - ``mongodb_atlas_whitelistip``
+       - IP address or CIDR block from which your |service| cluster is
+         accessible.
+
+  To maximize security, consider taking the following steps: 
```

Comment:
This is an optional but highly encouraged procedure. Maybe we can delineate both of those by putting it in an admonition? We could probably use either `class: note` or `class: admonition-example` styling.

```rst
.. admonition:: Use an Input Variables File to Maximize Security
   :class: admonition-example
   
   To maximize security, consider taking the following steps:
   
   ...
```

Up to you on if we should do this